resync/core/context_store.py

```
@@ -53,7 +53,7 @@ async def add_conversation(
         )
 
     # Sync wrapper for backward compatibility
-    def add_conversation_sync(self, session_id: str, role: str, content: str) -> None:
+    def add_conversation_sync(self, *_args: Any, **_kwargs: Any) -> None:
```

Ruff (0.15.0)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `*_args`

(ANN401)